tests/test_log.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+
+import logging
+
+import pytest
+
+from cookiecutter.log import configure_logger
+
+
+def create_log_records():
+    cookiecutter_logger = logging.getLogger('cookiecutter')
+    foo_logger = logging.getLogger('cookiecutter.foo')
+    foobar_logger = logging.getLogger('cookiecutter.foo.bar')
+
+    cookiecutter_logger.info('Welcome to Cookiecutter')
+    cookiecutter_logger.debug('Generating project from pytest-plugin')
+    foo_logger.info('Loading user config from home dir')
+    foobar_logger.debug("I don't know.")
+    foobar_logger.debug('I wanted to save the world.')
+    foo_logger.error('Aw, snap! Something went wrong')
+    cookiecutter_logger.debug('Successfully generated project')
+
+
+@pytest.fixture
+def info_messages():
+    return [
+        'INFO: Welcome to Cookiecutter',
+        'INFO: Loading user config from home dir',
+        'ERROR: Aw, snap! Something went wrong',
+    ]
+
+
+@pytest.fixture
+def debug_messages():
+    return [
+        'INFO cookiecutter: '
+        'Welcome to Cookiecutter',
+
+        'DEBUG cookiecutter: '
+        'Generating project from pytest-plugin',
+
+        'INFO cookiecutter.foo: '
+        'Loading user config from home dir',
+
+        "DEBUG cookiecutter.foo.bar: "
+        "I don't know.",
```

Comment:
``don't`` 😿 

I think double quotes are better than escaping the single quote.